packages/back-end/src/controllers/features.ts

```diff
@@ -506,7 +508,9 @@ export async function postFeatureRebase(
   const environments = filterEnvironmentsByFeature(allEnvironments, feature);
   const environmentIds = environments.map((e) => e.id);
 
-  req.checkPermissions("manageFeatures", feature.project);
+  if (!context.permissions.canUpdateFeature(feature, {})) {
```

Comment:
We are using this pattern a lot in this file - where we're checking if they have permission to update the feature, but it's changing an element of the feature that isn't something that if changed, would change their permission level. Open to ideas if this pattern is confusing to others.